tests/integration/conftest.py

```diff
@@ -318,7 +318,7 @@ async def deploy_traefik_if_not_deployed(ops_test: OpsTest, traefik_charm):
 
     # now we're most definitely active.
     async with ops_test.fast_forward():
-        await ops_test.model.wait_for_idle(["traefik-k8s"], status="active", timeout=1000)
+        await ops_test.model.wait_for_idle(["traefik-k8s"], timeout=1000)
```

Comment:
I'm surprised this ever passed: fast-forward would emit update-status every 5 sec so the charm will never be idle for the [default 15sec](https://github.com/juju/python-libjuju/blob/d809caf11dc2e16fd2dc2821383a8a452c634a9e/juju/model.py#L2472).

Reply:
that is also a good point

Reply:
but that line of code is in a lot of places, why did it only fail there?

Reply:
maybe that's why they were flaky: if it happens to check the status while it's performing an update, it will get maintenance, otherwise it will get active

Reply:
I'm getting rid of all fast forwards to see what happens

Reply:
maybe it would also work to remove the 'active' constraint only